graphwalker-core/src/main/java/org/graphwalker/core/model/Edge.java

```diff
@@ -39,281 +39,304 @@
 import static org.graphwalker.core.model.Vertex.RuntimeVertex;
 
 /**
- * <h1>Edge</h1>
- * The  Edge holds the information for a transition in a model.
+ * <h1>Edge</h1> The Edge holds the information for a transition in a model.
  * </p>
  * The edge represents an action performed by a test, which takes the system
- * under test, from a state to another.
- * The edge has a source and target vertex. If the vertices are identical, the
- * edge is a self loop. The source vertex is not mandatory, but in a model,
- * there should be only one such instance. Also, the target vertex is not
- * mandatory, but again, in a model, there should be only one such instance.
+ * under test, from a state to another. The edge has a source and target vertex.
+ * If the vertices are identical, the edge is a self loop. The source vertex is
+ * not mandatory, but in a model, there should be only one such instance. Also,
+ * the target vertex is not mandatory, but again, in a model, there should be
+ * only one such instance.
  *
  * @author Nils Olsson
  */
 public final class Edge extends CachedBuilder<Edge, Edge.RuntimeEdge> {
 
-  private Vertex sourceVertex;
```

Comment:
The diff here is way larger then it is in reality. My guess it might have to do with line endings?
Anyhow, this makes is hard to see what the actual change is.
Could you fix this?